airflow/hooks/oracle_hook.py

```diff
@@ -50,20 +50,61 @@ def get_conn(self):
         as in ``{ "dsn":"some.host.address" , "service_name":"some.service.name" }``
         """
         conn = self.get_connection(self.oracle_conn_id)
+        conn_config = {
+            'user': conn.login,
+            'password': conn.password or ''
```

Comment:
Is empty string different to None? I'd probably say the `or ''` isn't needed, but not certain with Oracle.

Reply:
You're right, my fault.